docs/index.md

```diff
@@ -0,0 +1,25 @@
+# Wagtail XLIFF Translation
+
+*Wagtail XLIFF Translation* is a Wagtail library for converting a Wagtail website to XLIFF (XML Localization Interchange File Format) and back. 
```

Comment:
Maybe specify it can be used for converting a sub-tree of the website (or even a single page) to xliff as well.